archive/app/controllers/ArchiveController.scala

```diff
@@ -59,6 +60,7 @@ object ArchiveController extends Controller with Logging with ExecutionContexts
         Some(path)
       case _ => None
     }
+    s"http://${normalised.getOrElse(path)}"
   }
```

Comment:
It's the same as before, another part of the controller was prefixing `http` to the normalised path. I decided to move all the string manipulation into `normalise`